src/bun.js/api/server.zig

```diff
@@ -91,6 +91,58 @@ const SendfileContext = struct {
 const DateTime = bun.DateTime;
 const linux = std.os.linux;
 
+const BlobFileContentResult = struct {
+    data: [*c]const u8,
+
+    const log = Output.scoped(.BlobFileContentResult, false);
+    fn init(comptime fieldname: []const u8, js_obj: JSC.JSValue, global: *JSC.JSGlobalObject, exception: JSC.C.ExceptionRef) ?BlobFileContentResult {
+        if (JSC.WebCore.Body.Value.fromJS(global, js_obj)) |body| {
+            if (body == .Blob and body.Blob.store != null and body.Blob.store.?.data == .file) {
+                const pathlike = body.Blob.store.?.data.file.pathlike;
```

Comment:
This entire function should be removed and replaced with a version of `readFile` easier to call from Zig,  like `writeFileWithPathBuffer`.

https://github.com/oven-sh/bun/blob/1973975ce0f56cea236697fcab36d8e2a211ac70/src/bun.js/node/node_fs.zig#L3290-L3428

Reply:
done on https://github.com/oven-sh/bun/pull/2934/commits/5d2830353e60d42f869b4b0e8571c9f763b473f3